.github/workflows/core.yml

```diff
@@ -196,6 +196,7 @@ jobs:
           submodules: recursive
       - uses: ./.github/actions/environment
       - run: nix-shell --run "poetry run make -C core build_unix"
+      - run: nix-shell --run "cd vendor/ts-tvl/tvl/server/model_config && poetry install && poetry run model_server tcp -c model_config.yml &"
```

Comment:
I think `poetry install` can be omitted. The python environment should already be installed at this point.

Reply:
> I think `poetry install` can be omitted

I tried and it doesn't seem to work.

Reply:
problem is that `ts-tvl` has its own pyproject.toml so as far as `poetry run` is concerned it's a separate environment.
ideally we should avoid the `cd vendor/...` thing, because `model_server` is available to us from root -- but @ibz tells me that the model is looking for some files in cwd... personally i'd keep it simple, keep the `poetry install`, and we have a different method of launching the model in the pipeline anyway
(emulator will launch it itself somewhere around `tropic_init()`)